src/modules/auth/auth.service.ts

```diff
@@ -35,6 +36,27 @@ export class AuthService {
     return { accessToken };
   }
 
+  async login(loginDto: LoginDto) {
+    const user = await this.usersRepository.findOne({
+      where: { email: loginDto.email },
+    });
+
+    if (!user) {
+      throw new BadRequestException('Invalid email or password');
+    }
+
+    const isPasswordValid = await Hasher.verifyHash(
+      user.passwordHash,
+      loginDto.password,
+    );
+    if (!isPasswordValid) {
+      throw new BadRequestException('Invalid email or password');
+    }
+    const payload = { sub: user.id, userEmail: user.email };
+    const accessToken = await this.generateAccessToken(payload);
+
+    return { accessToken };
+  }
   async generateAccessToken(payload: { sub: string; userEmail: string }) {
     const accessToken = await this.jwtService.signAsync(payload, {
       secret: this.configService.get('AUTH_ACCESS_TOKEN_SECRET'),
```

Comment:
After providing expiresIn and secret options inside auth.module.ts during import, you can omit options and just use 
```js
    const accessToken = await this.jwtService.signAsync(payload);
```
when you are creating access tokens.